### ISSUE TITLE
Error while trying to run calibration on remote client

### ISSUE THREAD

**Issue by Pyr47** [eoyilmaz/displaycal-py3#282]
We are running the remote calibration using Resolve on the remote machine. The process connects over the network just fine, and will start running, but then quickly seems to stop (no more screen pattern updates - but the shutter sound still plays, and generally the lines below repeat over and over). 

Both machine are Macs.

```
File "./.wait", line 4

    print(*["\nCurrent RGB"] + sys.argv[1:])
          ^

SyntaxError: invalid syntax

dispread: Warning - System command '/usr/bin/env python ./.wait 0 0 0 0.000000 0.000000 0.000000' failed with 256
```

As far as I can figure, this syntax appears to be correct. Anyone else run into this?

**Comment by mcorrigan** [comment]
+1

**Comment by eoyilmaz** [comment]
The syntax is correct, but with fb3fd4e we have improved it a little. Also we fixed the patches getting frozen when profiling remotely through Resolve issue with #302 , and this will be released with 3.9.12 👍

**Comment by eoyilmaz** [comment]
3.9.12 is released please try it.
